README.md

```diff
@@ -570,6 +573,26 @@ Error: ER_ACCESS_DENIED_ERROR: Access denied for user 'root'@'localhost' (using
 
 For best results we recommend using the default port configuration whenever possible.
 
+### I've upgraded WP Local Docker from a previous version and now the wpsnapshots command isn't working
+
+This is likely to do with the change from the [WP Snapshots](https://github.com/10up/wpsnapshots) system, to the new [Snapshots](https://github.com/10up/snapshots) WP-CLI command.
+This update removed the need for a dedicated container for the snapshots system, and instead uses the `phpfpm` container.
+Because of this, we need to ensure that the `phpfpm` container has access to the `~/.wpsnapshots` and `~/.aws` directories on your host machine.
+
+To provide that access, please edit the `docker-compose.yml` file in the environment you are updating, to add the following lines:
+
+```yaml
+  phpfpm:
+      image: '10up/wp-php-fpm-dev:7.4-ubuntu'
+      # ...
+      volumes:
+         # ...
+          - '~/.wpsnapshots:/home/www-data/.wpsnapshots:cached'
+          - '~/.aws:/home/www-data/.aws:cached'
```

Comment:
might be safer to mount ~/.aws as read only by adding :ro to the end of that `~/.aws:/home/www-data/.aws:cached:ro`.